config/operator/namespace/cluster_role.template.yaml

```diff
@@ -90,19 +94,6 @@ rules:
   - update
   - patch
   - delete
-- apiGroups:
-  - associations.k8s.elastic.co
-  resources:
-  - apmserverelasticsearchassociations
-  - apmserverelasticsearchassociations/status
-  verbs:
-  - get
-  - list
-  - watch
-  - create
-  - update
-  - patch
-  - delete
```

Comment:
Why can we delete this RBAC rule?

Reply:
Because the corresponding CRD does not exist anymore?

Reply:
Of course. :confused: